### PR TITLE
Added Avocado Prices data

### PR DESCRIPTION
**Pull request by evejcik**

### CODE REVIEW

**Comment by beanumber** [comment]
Working on issue #4

**Comment by beanumber** [comment]
@evejcik Thanks for starting this PR. Can you add the others mentioned in #4 as collaborators on your fork?

File: data-raw/process_data_sets.R [hunk 1]
```diff
@@ -272,3 +272,12 @@ pennies_resamples <-
...
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+# Avocado Prices By Region
+# Original Google Sheet here:
+# https://docs.google.com/spreadsheets/d/1cNuj9V-9Xe8fqV3DQRhvsXJhER3zTkO1dSsQ1Q0j96g/edit#gid=1419070688
+avocado_prices <-
+  "https://wjhopper.github.io/SDS-201/data/avocados.csv" %>%
+  read_csv() 
+usethis::use_data(pennies_resamples, overwrite = TRUE)
```
Comment (beanumber) [inline data-raw/process_data_sets.R]: `pennies_resamples`???

File: R/datasets.R [hunk 2]
```diff
@@ -362,3 +362,25 @@
...
+#' Gathered from <https://docs.google.com/spreadsheets/d/1cNuj9V-9Xe8fqV3DQRhvsXJhER3zTkO1dSsQ1Q0j96g/edit#gid=1419070688>
+#' @format A data frame of 54 regions over 3 years of weekly results
+#' \describe{
+#' \item{Date}{Week of Data Recording}
+#' \item{Average Price}{Average Price of Avocado}
+#' \item{Total Volume}{Total Amount of Avocados}
+#' \item{Small Haas Sold}{Amount of Small Haas Avocados Sold}
+#' \item{Large Haas Sold}{Amount of Large Haas Avocados Sold}
+#' \item{XLarge Haas Sold}{Amount of Extra Large Haas Avocados Sold}
+#' \item{Total Bags}{Total Amount of Bags of Avocados}
```
Comment (beanumber) [inline R/datasets.R]: These variable names have spaces and are not in snake_case. Can you please run `janitor::clean_names()` on them?

Comment (tessgold) [reply R/datasets.R]: added janitor::clean_names to process_data_sets.R and changed names here

File: data-raw/process_data_sets.R (hunk 1 above)
Comment (tessgold) [reply data-raw/process_data_sets.R]: changed name to avacados

File: NEWS.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+#moderndive 0.5.3.9002
```
Comment (mariumtapal) [inline NEWS.md]: Version number is inconsistent with the one in the DESCRIPTION file. 9002 vs 9003.

File: R/datasets.R [hunk 4]
```diff
@@ -362,3 +362,25 @@
 #' }
 #' @seealso This data was sampled from the `movies` data frame in the `ggplot2movies` package.
 "movies_sample"
+
+
+#' Avocado Prices by US Region
+#' 
+#' Gathered from <https://docs.google.com/spreadsheets/d/1cNuj9V-9Xe8fqV3DQRhvsXJhER3zTkO1dSsQ1Q0j96g/edit#gid=1419070688>
+#' @format A data frame of 54 regions over 3 years of weekly results
```
Comment (mariumtapal) [inline R/datasets.R:371]: Please add a blank line before the `@format` line

**Comment by mariumtapal** [comment]
The `type` variable in the dataset is of type `chr` right now. Because there are only 2 types of avocados, it makes sense to make this variable into a factor.

File: data-raw/process_data_sets.R [hunk 5]
```diff
@@ -281,4 +281,5 @@ avocados <-
   "https://wjhopper.github.io/SDS-201/data/avocados.csv" %>%
   read_csv() %>%
   janitor::clean_names()
+avocados$type <- as.factor(avocados$type)
```
Comment (mariumtapal) [inline data-raw/process_data_sets.R]: Please use `mutate()` to do this

**Review by mariumtapal — CHANGES_REQUESTED** [review]

File: data-raw/process_data_sets.R [hunk 6]
```diff
@@ -272,3 +272,14 @@ pennies_resamples <-
...
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+# Avocado Prices By Region
+# Original Google Sheet here:
+# https://docs.google.com/spreadsheets/d/1cNuj9V-9Xe8fqV3DQRhvsXJhER3zTkO1dSsQ1Q0j96g/edit#gid=1419070688
+avocados <-
+  "https://wjhopper.github.io/SDS-201/data/avocados.csv" %>%
+  read_csv() %>%
+  janitor::clean_names()
```
Comment (mariumtapal) [inline data-raw/process_data_sets.R]: You can remove the explicit call to `janitor::` since we load it at the beginning of the script

**Review by mariumtapal — CHANGES_REQUESTED** [review]

File: NEWS.md [hunk 7]
```diff
@@ -1,3 +1,9 @@
+#moderndive 0.5.3.9003
```
Comment (beanumber) [inline NEWS.md]: Make the next version in the NEWS 0.5.4.

File: data-raw/process_data_sets.R [hunk 8]
```diff
@@ -272,3 +272,15 @@ pennies_resamples <-
...
+# Avocado Prices By Region
+# Original Google Sheet here:
+# https://docs.google.com/spreadsheets/d/1cNuj9V-9Xe8fqV3DQRhvsXJhER3zTkO1dSsQ1Q0j96g/edit#gid=1419070688
+avocados <-
+  "https://wjhopper.github.io/SDS-201/data/avocados.csv" %>%
+  read_csv() %>%
+  clean_names() %>%
+  mutate(type = as.factor(type)) %>%
+  rename(xlarge_hass_sold = xlarage_hass_sold)
+usethis::use_data(avocados, overwrite = TRUE)
```
Comment (beanumber) [inline data-raw/process_data_sets.R:286]: Why are you changing all of these other data files? Please delete these changes from the PR.

**Review by beanumber — CHANGES_REQUESTED** [review]

File: R/datasets.R [hunk 9]
```diff
@@ -362,3 +362,26 @@
...
+#' Gathered from <https://docs.google.com/spreadsheets/d/1cNuj9V-9Xe8fqV3DQRhvsXJhER3zTkO1dSsQ1Q0j96g/edit#gid=1419070688>
+#' 
+#' @format A data frame of 54 regions over 3 years of weekly results
+#' \describe{
+#' \item{date}{Week of Data Recording}
+#' \item{average_price}{Average Price of Avocado}
+#' \item{total_volume}{Total Amount of Avocados}
+#' \item{small_hass_sold}{Amount of Small Haas Avocados Sold}
+#' \item{large_hass_sold}{Amount of Large Haas Avocados Sold}
+#' \item{xlarage_hass_sold}{Amount of Extra Large Haas Avocados Sold}
```
Comment (beanumber) [inline R/datasets.R]: `x_larage`? Is this a typo?

File: NEWS.md [hunk 10]
```diff
@@ -1,3 +1,9 @@
+# #moderndive 0.5.4
```
Comment (beanumber) [inline NEWS.md]: Please remove the extra `#`

**Review by beanumber — CHANGES_REQUESTED** [review]

**Review by beanumber — APPROVED** [review]
Great work!

**Pull request merged**